backend/prisma/seed.js

```diff
@@ -504,6 +509,136 @@ async function main() {
     },
   });
 
+  console.log('Creating interaction configs...');
+  const cse210_interaction_config = await prisma.interaction_configs.create({
+    data: {
+      course_id: cse210.id,
+      config: {
+        text: 'Record an interaction with students',
+        options: ['positive', 'negative', 'neutral'],
+      },
+    },
+  });
+
+  const cse110_interaction_config = await prisma.interaction_configs.create({
+    data: {
+      course_id: cse110.id,
+      config: {
+        text: 'Record an interaction with students',
+        options: ['positive', 'negative', 'neutral'],
```

Comment:
[Suggestion]
Each option can have a `value` and a `color`. The color will essentially help bring consistent UI colors for the same option on the frontend.